### PR TITLE
update route and page

### PR DESCRIPTION
**Pull request by trietpham252**

### CODE REVIEW

**Comment by trietpham252** [comment]
Screenshot 2023-12-13 at 20 31 13
Screenshot 2023-12-13 at 20 31 40
Screenshot 2023-12-13 at 20 31 50

File: admin/src/App.tsx
Comment (llamas-laptrinh) [inline admin/src/App.tsx:1]: move to router folder

File: admin/src/pages/HomePage/HomePage.tsx
Comment (llamas-laptrinh) [inline admin/src/pages/HomePage/HomePage.tsx:1]: rename

File: admin/src/pages/Tabs/Tabs.tsx
Comment (llamas-laptrinh) [inline admin/src/pages/Tabs/Tabs.tsx:1]: tabs is components

File: admin/src/routes/index.tsx
Comment (llamas-laptrinh) [inline admin/src/routes/index.tsx:1]: move to router

**Comment by trietpham252** [comment]
Screenshot 2023-12-13 at 21 32 40

**Comment by trietpham252** [comment]
Screenshot 2023-12-13 at 23 59 39

**Pull request merged**